modules/tinymce/src/core/main/ts/api/Events.ts

```diff
@@ -3,62 +3,89 @@ import { FormatVars } from '../fmt/FormatTypes';
+const firePostProcess = (editor: Editor, args: ParserArgs & { content: string }): EditorEvent<PostProcessEvent> =>
+  editor.dispatch('PostProcess', args);
 
-const fireRemove = (editor: Editor) => editor.dispatch('remove');
+const fireRemove = (editor: Editor): void => {
+  editor.dispatch('remove');
+};
 
-const fireDetach = (editor: Editor) => editor.dispatch('detach');
+const fireDetach = (editor: Editor): void => {
+  editor.dispatch('detach');
+};
 
-const fireSwitchMode = (editor: Editor, mode: string) => editor.dispatch('SwitchMode', { mode });
+const fireSwitchMode = (editor: Editor, mode: string): void => {
+  editor.dispatch('SwitchMode', { mode });
+};
 
-const fireObjectResizeStart = (editor: Editor, target: HTMLElement, width: number, height: number, origin: string) => {
+const fireObjectResizeStart = (editor: Editor, target: HTMLElement, width: number, height: number, origin: string): void => {
   editor.dispatch('ObjectResizeStart', { target, width, height, origin });
 };
 
-const fireObjectResized = (editor: Editor, target: HTMLElement, width: number, height: number, origin: string) => {
+const fireObjectResized = (editor: Editor, target: HTMLElement, width: number, height: number, origin: string): void => {
   editor.dispatch('ObjectResized', { target, width, height, origin });
 };
 
-const firePreInit = (editor: Editor) => editor.dispatch('PreInit');
+const firePreInit = (editor: Editor): void => {
+  editor.dispatch('PreInit');
+};
 
-const firePostRender = (editor: Editor) => editor.dispatch('PostRender');
+const firePostRender = (editor: Editor): void => {
+  editor.dispatch('PostRender');
+};
 
-const fireInit = (editor: Editor) => editor.dispatch('Init');
+const fireInit = (editor: Editor): void => {
+  editor.dispatch('Init');
+};
 
-const firePlaceholderToggle = (editor: Editor, state: boolean) => editor.dispatch('PlaceholderToggle', { state });
+const firePlaceholderToggle = (editor: Editor, state: boolean): void => {
+  editor.dispatch('PlaceholderToggle', { state });
+};
 
-const fireError = (editor: Editor, errorType: string, error: { message: string }) => editor.dispatch(errorType, error);
+const fireError = (editor: Editor, errorType: string, error: { message: string }): void => {
+  editor.dispatch(errorType, error);
+};
 
-const fireFormatApply = (editor: Editor, format: string, node: Node | RangeLikeObject, vars: FormatVars | undefined) =>
+const fireFormatApply = (editor: Editor, format: string, node: Node | RangeLikeObject | null | undefined, vars: FormatVars | undefined): void => {
   editor.dispatch('FormatApply', { format, node, vars });
+};
 
-const fireFormatRemove = (editor: Editor, format: string, node: Node | RangeLikeObject, vars: FormatVars | undefined) =>
+const fireFormatRemove = (editor: Editor, format: string, node: Node | RangeLikeObject | undefined, vars: FormatVars | undefined): void => {
```

Comment:
```suggestion
const fireFormatRemove = (editor: Editor, format: string, node: Node | RangeLikeObject | null | undefined, vars: FormatVars | undefined): void => {
```
Should match the above function?